### ISSUE TITLE
Validation is not supported

### ISSUE THREAD

**Issue by markwpiper** [Toilal/ng-pickadate#7]
Using Angular validation on a pickatime input (and I suspect also pickadate input) does not work.  The input is not validated and the validation CSS styles are not applied.

eg: the following field should be required but in practice is not

```
<input type="text" name="deliveryTime" pick-a-time="data.deliveryTime" required>
```

I think this occurs because pickadate would require a custom validator.  See Custom Validation
 in https://docs.angularjs.org/guide/forms

**Comment by Toilal** [comment]
I did not try by myself, but i'm surprised because the directive doesn't change input, it's still a standard `<input type="text" ...>` so it should support default validation based on text value.

Is your datepicker included in a `<form>` ? (it is required for validation)

**Comment by markwpiper** [comment]
Yes, it's included in a form. I noticed that if an ng-model is also set on the input, then the validation works as expected; however, the user can only set the value once.  (Picking a time once works, but going back to the widget to change the time has no effect.)

eg:
<input type="text" name="deliveryTime" ng-model="data.deliveryTime" pick-a-time="data.deliveryTime" required>

**Comment by Toilal** [comment]
Ho yes, true I also hit this one time, the input must be binded for
validation to work.

**Comment by Toilal** [comment]
It's working properly in the latest released version (v0.2.0), and with example on the website : http://toilal.github.io/ng-pickadate/
